### DISCUSSION TITLE
How Custom Capacitor Plugin reads configuration from `capacitor.config.json`?

### DISCUSSION THREAD

**Discussion by thanhhoa214** [ionic-team/capacitor#4408] [Ideas]
I'm working on my custom plugin and the need for reading the initial configuration? Hope your help guys, thanks in advance.👍

**Comment by jcesarmobile** [comment]
see https://github.com/ionic-team/capacitor/discussions/4355#discussioncomment-493699

**Reply by thanhhoa214** [reply]
But can I retrieve it from `web.ts`?

**Reply by thanhhoa214** [reply]
In case I wanna ask for `appId` & `secretKey`, pass it through parameters should be avoided at all?
